dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1235,6 +1278,21 @@ winrt::TeachingTipPlacementMode TeachingTip::DetermineEffectivePlacement()
             bool leftCenterAvailable = true;
             bool leftTopAvailable = true;
             bool leftBottomAvailable = true;
+            auto screenSize = winrt::Size({ 0.0f, 0.0f });
+            if (!shouldConstrainToRootBounds)
+            {
```

Comment:
I was expecting an else clause here where this thing calculated the bounds. I'm confused why this is only calculating screenSize and then the things below sometimes use windowBounds and sometimes use screenBounds. Can we just produce one local that's the constraintBounds and then the logic below doesn't have to look at the `shouldConstraintToRootBounds` variable again?